src/ragas/dataset_schema.py

```diff
@@ -590,6 +617,81 @@ def from_json(cls, path, metric_name: t.Optional[str]) -> "MetricAnnotation":
             }
         )
 
+    @classmethod
+    def from_json(cls, path: str, metric_name: t.Optional[str]) -> "MetricAnnotation":
+        """Load annotations from a JSON file"""
+        dataset = json.load(open(path))
+        return cls._process_dataset(dataset, metric_name)
+
+    @classmethod
+    def from_ragas_platform(
```

Comment:
this brings to the point about what we are calling the thing at app.ragas.io - IMO is should be Ragas App and so the method should be `from_ragas_app`

Reply:
from_app is what I have named it right now, in case we need to be explicit, lets call it from_ragas_app